docker-compose.yaml

```diff
@@ -58,6 +58,20 @@ services:
       interval: 30s
       timeout: 10s
       retries: 5
+  archive-mock:
+    build: https://github.com/SwissOpenEM/ScicatArchiveMock.git
+    depends_on:
+      rabbitmq:
```

Comment:
just a thought. This is already covered by the dependency to the backend. I think it makes sense anyway to have it here, as it is indeed a direct dependency. 

I am happy to keep it like this if you think so

Reply:
I think I'll leave it, it represents a bit better the dependencies.